components/brave_rewards/browser/test/common/rewards_browsertest_contribution.cc

```diff
@@ -454,6 +454,8 @@ void RewardsBrowserTestContribution::SetUpUpholdWallet(
     const double balance,
     const ledger::type::WalletStatus status) {
   DCHECK(rewards_service);
+  browser_->profile()->GetPrefs()->SetString(
```

Comment:
where are the gemini tests? Also can you fix this test name?

Reply:
added test for gemini here - https://github.com/brave/brave-core/pull/8566/files#diff-4968fc4cfac5fb8ee0708f8dd8f6fdca41739502de65a023a27f5fbe53b61e14R320

Created a follow-up issue to rename this file - https://github.com/brave/brave-browser/issues/16800 - Since, it is used by other contribution browsertests I think a separate PR to address this would make sense.